newIDE/electron-app/app/DebuggerServer.js

```diff
@@ -1,5 +1,5 @@
-const WebSocket = require('ws');
-const log = require('electron-log');
+const WebSocket = require("ws");
```

Comment:
looks like prettier has added some noise to your diff due to your settings being different from those of the project

Reply:
@blurymind sure I won't mind removing the double commas